### ISSUE TITLE
dev.mod - Signal button clicking second time make it stuck

### ISSUE THREAD

**Issue by madhurrya** [sartography/spiff-arena#1368]
In the Create Event model I added data and in the next page I clicked edit and came back to the first form and edited data and again clicked Edit in the next page. Now the page is stuck.
https://dev.mod.spiff.status.im/process-models/comms-and-brand:event-management:event-execution:create-event

![Image](https://github.com/sartography/spiff-arena/assets/110527567/35b7cc72-af1d-49c5-877e-64260b6e776d)

Managed to recreate the same using this simple model.
https://dev.app.spiff.status.im/process-models/misc:qa:forms:simple-form-name-age

![Image](https://github.com/sartography/spiff-arena/assets/110527567/58b6884d-eb79-4ed0-a17c-a5c332d9fbed)

**Comment by burnettk** [comment]
The issue always happens with the simple model, or just sometimes? If always, how do you repro?

**Comment by madhurrya** [comment]
Yes, enter your name and continue and click edit on the show name page. Do the same again. When you click Edit this time it get stuck.

**Comment by madhurrya** [comment]
Oops I have shared the link of the dev.app model which is working. This is what is not working. The dev.mod model
https://dev.mod.spiff.status.im/process-models/misc:qa:forms:simple-form-name-age

**Comment by jasquat** [comment]
This seems to be related to the comment here https://github.com/sartography/spiff-arena/issues/1861#issuecomment-2206467374.

I think the correct thing to do is add a validation in bpmn-js-spiffworkflow telling users that this is invalid. A task cannot have multiple inputs. They must be merged with an exclusive gateway at least.
